tests/Unit/ParallelAlgorithms/DiscontinuousGalerkin/Test_InitializeInterfaces.cpp

```diff
@@ -138,8 +170,67 @@ void check_compute_items(
       domain::Tags::Interface<
           domain::Tags::BoundaryDirectionsExterior<Dim>,
           ::Tags::Normalized<domain::Tags::UnnormalizedFaceNormal<Dim>>>{}));
+
+  if (UseMovingMesh) {
+  }
```

Comment:
If this empty `if` is here for an important reason, a comment would be helpful. Else, can it be removed?